Automattic-Tracks-iOS/Crash Logging/CrashLogging.swift

```diff
@@ -94,6 +120,10 @@ public class CrashLogging {
     public static func crash() {
         Client.shared?.crash()
     }
+
+    public static var eventLogging: EventLogging? {
```

Comment:
Yep! It's just to allow the hosting application to read state out of it – it can only be set and shouldn't be allowed to be changed after initialization – if you want a new one I think it's pretty reasonable to require a new `CrashLogging` stack

Reply:
That was my not knowing Swift thing coming through. It's already a getter only 🤦